student-test/assignment1/Test_validate_username.c

```diff
@@ -18,5 +18,7 @@ void test_validate_my_username()
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+    //TEST_ASSERT_TRUE_MESSAGE(false,"mubeena12");
+    char *actual_name = malloc_username_from_conf_file();
+    TEST_ASSERT_EQUAL_STRING_MESSAGE("mubeena12", actual_name, "AESD student");
```

Comment:
@mubeena12 what cleanup is needed for `actual_name` here?  When will the print for `AESD student` be printed?